tensorflow/python/autograph/operators/py_builtins_test.py

```diff
@@ -293,6 +293,34 @@ def test_filter_dataset(self):
       self.assertAllEqual(self.evaluate(iterator.get_next()), 2)
       self.assertAllEqual(self.evaluate(iterator.get_next()), 1)
 
+  def test_any(self):
+    self.assertEqual(
+        py_builtins.any_([False, True, False]), True)
+    self.assertEqual(
+        py_builtins.any_([False, False, False]), False)
+
+  def test_any_dataset(self):
+    dataset_1 = dataset_ops.DatasetV2.from_tensor_slices([False, True, False])
+    dataset_2 = dataset_ops.DatasetV2.from_tensor_slices([False, False, False])
+    self.assertEqual(
+        py_builtins.any_(dataset_1), True)
```

Comment:
Looking at the test results, it seems that this needs a `self.evaluate`.

Reply:
Thanks @mdanatg, PR Updated.